Neos.Workspace.Ui/Classes/Controller/WorkspaceController.php

```diff
@@ -136,85 +158,88 @@ public function indexAction(): void
         $this->view->assign('displayContentRepositorySelector', $numberOfContentRepositories > 1);
```

Comment:
this selector, and the ability to choose between content repositories, got lost (see https://github.com/neos/neos-development-collection/pull/5132/#issuecomment-2447319888)

we can probably live without this. Though we then should fully remove the unused variables?